csharp/ql/test/library-tests/statements/Lock3.ql

```diff
@@ -0,0 +1,8 @@
+import csharp
+
+from Method m, LockStmt ls, Expr lockExpr
+where
+  ls.getEnclosingCallable() = m and
+  m.getName() = "LockMethod" and
```

Comment:
Minor: we probably don't need to filter the statements to be inside this method.